src/common/utils/hooks/useAssetHub.ts

```diff
@@ -1,24 +1,24 @@
+import type { GetTransferFee } from './types';
+
 import { formatBalance } from '../balance';
 import { ZERO_BALANCE } from '../constants';
 
-import { type Asset } from '@/common/chainRegistry/types';
+import type { Asset } from '@/common/chainRegistry/types';
 import { TransactionType, useExtrinsic } from '@/common/extrinsicService';
 import { useQueryService } from '@/common/queryService/QueryService';
-import { type ChainId } from '@/common/types';
-
-import { type GetAssetHubGiftFee, type GetTransferFee } from './types';
+import type { ChainId } from '@/common/types';
 
 const DOT_ASSET_ID = '0';
 
 export const useAssetHub = () => {
-  const { handleFee } = useExtrinsic();
+  const { getTransactionFee } = useExtrinsic();
   const { assetConversion, getExistentialDeposit, getFreeBalanceStatemine } = useQueryService();
 
-  const getTransferFee: GetTransferFee = async ({ chainId, assetId, transferAmmount, address }) => {
-    const transferDotFee = await handleFee(
+  const getTransferFee: GetTransferFee = async ({ chainId, assetId, transferAmount, address }) => {
+    const transferDotFee = await getTransactionFee(
       chainId as ChainId,
       TransactionType.TRANSFER_STATEMINE,
-      transferAmmount,
+      transferAmount,
       assetId,
     );
     // TODO: Delete it after PR for asset hub is merged
```

Comment:
to avoid ED AH fee for USDT we can delete this part now if it was updated on the asset hub side (need to test it first)
And then change the way we calculate `totalDotFee` and `giftDotFee`